src/components/ProgramRecordsList.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+
+import { ChevronLeft, Info } from '@edx/paragon/icons';
+import { Alert, Hyperlink } from '@edx/paragon';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
+import { getConfig } from '@edx/frontend-platform/config';
+import { logError } from '@edx/frontend-platform/logging';
+import _ from 'lodash';
+import getProgramRecords from './data/service';
+
+const ProgramRecordsList = () => {
+  const renderProfile = () => {
+    const { username } = getAuthenticatedUser();
+    return (
+      <Hyperlink
+        destination={`${getConfig().LMS_BASE_URL}/u/${username}`}
+        className="mt-2 mb-4 d-inline-block muted-link pl-3 pr-3"
+      >
+        <ChevronLeft className="mb-1" />
+        <FormattedMessage
+          id="records.profile.link"
+          defaultMessage="Back to My Profile"
+          description="Link text that redirects logged-in user to their profile page"
+        />
+      </Hyperlink>
+    );
+  };
+
+  const renderCredentialsServiceIssueAlert = () => (
+    <div tabIndex="-1">
+      <Alert variant="danger">
+        <Info className="text-danger-500 mr-2 mb-1" />
+        <FormattedMessage
+          id="records.list.error"
+          defaultMessage="An error occurred attempting to retrieve your program records. Please try again later."
+          description="An error message indicating there is a problem retrieving the user's program records"
+        />
+      </Alert>
+    </div>
+  );
+
+  const renderEmpty = () => (
+    <p className="pl-3 pr-3">
+      <FormattedMessage
+        id="records.list.empty"
+        defaultMessage="No records yet. Program records are created once you have earned at least one course
+        certificate in a program."
+        description="A message indicating the user has no program records to display on the Learner Records page"
+      />
+    </p>
+  );
+
+  const [records, setRecords] = React.useState([]);
+
+  const renderRows = () => (
+    <section id="program-records-list" className="pl-3 pr-3">
+      <header>
+        <h3 className="h4">
```

Comment:
(Please see my other comment on header levels at line 111, this applies here as well.)